swagger/swagger.html

```diff
@@ -112,6 +113,45 @@
         label: function() {
             return this.summary || "swagger doc";
         },
+        onpaletteadd: function() {
+            //setup swagger-ui
+            var swaggerUi;
+            var refreshInfoHeight = 75;
+            var content = $("<div/>",{
+                id:"tab-swagger-ui", style:"position: relative; padding: 0px 4px; height: 100%; overflow: hidden"
+            });
+            var refreshInfo = $("<div/>", {
+                style:"height: "+ refreshInfoHeight + "px; margin: 5px; text-align: center"
+            });
+            $("<p/>", {
+                text: "Please click refresh after making changes to any HTTP nodes."
```

Comment:
Ordinarily, this text should be i18ned - but as it is a temporary thing until auto-refresh-on-deploy is enabled, we can let it stay for the moment.